lib/smart_answer_flows/marriage-abroad/outcomes/countries/jordan/ceremony_country/partner_british/_opposite_sex.govspeak.erb

```diff
@@ -2,63 +2,63 @@ Contact the relevant local authorities in Jordan to find out about local marriag
 
-You’ll need to complete a [‘Notice of marriage’](/government/publications/notice-of-marriage-form--2) form and an [‘Affidavit for marriage’](/government/publications/affirmationaffidavit-of-marital-status-form) form.
+If you or your partner have been divorced, widowed or previously in a civil partnership, you’ll also need whichever of the following documents is appropriate:
 
-You can download and fill in (but not sign) the forms in advance.
+- a decree absolute or final order
+- a civil partnership dissolution or annulment certificate
+- your (or your partner’s) former spouse or civil partner’s death certificate
 
-You must take the forms with you to your appointment.
+You’ll also need to provide evidence of nationality or residence if the divorce or dissolution took place outside the UK. You’ll need to get it legalised and translated if it’s not in English.
 
 %The names on all documents you provide must appear exactly as they do on your passports - if not, the authorities may refuse to allow the marriage to go ahead. You may need to provide evidence if the name on your passport is different to your birth certificate (for example marriage certificate or deed poll).%
 
-^Your partner will need to follow the same process and pay the fees to get their own CNI.^
-
-If you or your partner have been divorced, widowed or previously in a civil partnership, you’ll also need whichever of the following documents is appropriate:
-
-- a [single divorced declaration document](/government/publications/single-declaration-form-for-islamic-marriages-if-you-are-divorced-jordan) if you’re getting married at the Sharia Court
-- a [decree absolute or final order](/copy-decree-absolute-final-order)
-- a civil partnership dissolution or annulment certificate
-- your (or your partner’s) former spouse or civil partner’s [death certificate](/order-copy-birth-death-marriage-certificate/)
-
-^You’ll also need to provide evidence of nationality or residence if the divorce or dissolution took place outside the UK. You’ll need to get it [legalised](/get-document-legalised) and [translated](/government/collections/lists-of-translators-and-interpreters) if it’s not in English.^
+###If you can’t have a religious ceremony
 
-###What happens next
+If you can’t get married at church or Sharia Court, you may be able to get married at the British Embassy in Amman. You and your partner both need to be resident in Jordan and need to prove that there are no other suitable facilities to get married locally.  
 
-The embassy will charge a fee for taking the oath, and then display your notice of marriage publicly for a further 7 days. They’ll then issue the CNI, as long as nobody has registered an objection. There’s an additional fee for this.
+You’ll need to pay separate fees for this service.
 
-You should check if your CNI needs to be legalised (certified as genuine) by the local authorities in Jordan.
+Email <cons.amman@fco.gov.uk> for more information.
+           
+##Fees
```

Comment:
We should not be removing use of the partial. We should update the `marriage_abroad_services.yml` default and then re-instate.

This is the same for all of the outcomes.